templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/porter.yaml

```diff
@@ -132,17 +138,63 @@ uninstall:
         key: "{{ bundle.parameters.id }}"
 
 start:
-  - exec:
+  - terraform:
+      arguments:
+        - "output"
+      description: "Get VM hostname and rg from Terraform outputs"
```

Comment:
It's getting the resource_id now...

However can the API not pass the resource_id in as its an output from the user_resource deployment and stored in cosmos?

Happy to approve as is, but might be something to look at as would simplify things.

Reply:
Good spot, have updated the descriptions